content/6_Advanced/Catalan.mdx

```diff
@@ -186,7 +186,9 @@ def inverses():
 		inv[i - 1] = inv[i] * i % MOD
 
 
-def choose(n: int, r: int):
+def choose(n: int, r: int) -> int:
+	if n - r < 0:
```

Comment:
just n < r

Reply:
ok

Reply:
not changed?

Reply:
my bad, didn't paid attention to other file